tests/integration/valkey-benchmark.tcl

```diff
@@ -37,6 +37,20 @@ tags {"benchmark network external:skip logreqres:skip"} {
             default_set_get_checks
         }
 
+        test {benchmark: negative path} {
+            set wrong_host 127.0.0.0
```

Comment:
Hostnames/IPs and ports seem pretty basic; was there a specific reason to test these, or was it due to an edge case encountered or a broader fix?

Reply:
Hi @xbasel, Yes you are right, this is actually motivated by this PR https://github.com/valkey-io/valkey/pull/1029.